src/assets/cru-scss/_supernav.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:math';
```

Comment:
Same here, I don't see math being used. (maybe an unchanged line is using it, I only scrolled through the whole first file).